arkouda/pdarrayclass.py

```diff
@@ -911,7 +1103,7 @@ def prod(pda : pdarray) -> float:
     else:
         raise TypeError("must be pdarray {}".format(pda))
 
-def min(pda : pdarray) -> float:
+def min(pda : 'pdarray') -> float:
```

Comment:
could return a float64 or int64 depending on dtype of input pdarray

Reply:
fixed!